db/migrate/20210224145258_create_relations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRelations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :relations do |t|
```

Comment:
Same comment regarding naming - is `relations` too generic?

Reply:
ditto

Reply:
agreed, this is now `school_links`